test/quantization/fx/test_quantize_fx.py

```diff
@@ -2306,7 +2306,7 @@ def forward(self, x):
 
     def test_quantized_input_quantized_output(self):
```

Comment:
I think we can also add a test case similar to `test_quantized_input_quantized_multiple_output` for standalone modules